test/integration/test_path.py

```diff
@@ -101,3 +103,108 @@ def test_write_delete(backend_aware_bucketfs_params, children_poem, classic_poem
     poem_path1.rm()
     expected_names = {"classic", "highlands.txt"}
     assert _collect_all_names(poems_root) == expected_names
+
+
+def test_infer_path_onprem(backend, backend_aware_bucketfs_params):
+    """
+    Creates the PathLike and validates it.
+    """
+    if backend == "saas":
+        pytest.skip()
+    host_port = re.search(
+        r"http://(\d{1,3}(?:\.\d{1,3}){3}):(\d+)", backend_aware_bucketfs_params["url"]
+    )
+    url = infer_path(
+        bucketfs_host=host_port.group(1),
+        bucketfs_port=host_port.group(2),
+        bucketfs_name=backend_aware_bucketfs_params["service_name"],
+        bucket=backend_aware_bucketfs_params["bucket_name"],
+        bucketfs_user=backend_aware_bucketfs_params["username"],
+        bucketfs_password=backend_aware_bucketfs_params["password"],
+        path_in_bucket="onpremtest/",
+        bucketfs_use_https=backend_aware_bucketfs_params["verify"],
```

Comment:
This will fail if a backend other than onprem is selected.

Compare 
* [backend_aware_bucketfs_params](https://github.com/exasol/pytest-plugins/blob/main/pytest-backend/exasol/pytest_backend/__init__.py#L333)
* [backend_aware_saas_bucketfs_params](https://github.com/exasol/pytest-plugins/blob/main/pytest-backend/exasol/pytest_backend/__init__.py#L301)
*  [backend_aware_onprem_bucketfs_params](https://github.com/exasol/pytest-plugins/blob/main/pytest-backend/exasol/pytest_backend/__init__.py#L284)

all in pytest-plugins/pytest-backend

Reply:
OK - please excuse me. 
I probably was wrong. 
I didn't notice that you test case should only be executed onprem, but skipped on saas.

I now replaced fixture `backend` by `use_onprem` which should serve this purpose in my suggestion above.
Still, the test case needs to call `pytest.skip()` by himself.